app/controllers/candidate_interface/decisions_controller.rb

```diff
@@ -61,13 +61,14 @@ def confirm_withdraw
     end
 
     def withdrawal_feedback
+      flash[:success] = "You’ve withdrawn your application for #{@application_choice.current_course.name_and_code} at #{@application_choice.provider.name}"
```

Comment:
I wonder if we should drop this banner and add something after the reasons for withdrawal form has been submitted? (otherwise we'll end up with 2 banners?)

"Success" might also be an odd title in this context? (Too positive?)

Reply:
I've kept a "success" banner but i've moved it to after the candidate has submitted the feedback. wdyt?